src/object/icecrusher.cpp

```diff
@@ -48,8 +49,10 @@ IceCrusher::IceCrusher(const ReaderMapping& reader) :
   lefteye(),
   righteye(),
   whites(),
-  ic_size(NORMAL)
+  ic_size(NORMAL),
+  sideways(false)
```

Comment:
No need to initialize this to false. Bools are false upon initialization by default.

Reply:
Fixed.